### PR TITLE
Search bar was created

### PR DESCRIPTION
**Pull request by illiashcherbak**
- [DEMO LINK](https://illiashcherbak.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://illiashcherbak.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Comment by illiashcherbak** [comment]
Все тесты проходит, когда делаю npm test, но после pull request получаю ошибку run npm test

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,46 @@
...
+          type="search"
+          name="bigSearch"
+          class="input"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+        <img
+          src="./images/Search.svg"
+          alt="search"
+          class="search-icon"
+        >
```
Comment (Ihor-Karpyn) [inline src/index.html:32]: Better to use background image for decorative icon

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 900;
+  font-style: normal;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
```
Comment (Ihor-Karpyn) [inline src/style.css:17]:
```suggestion
html {
  font-family: "Avenir", sans-serif;
}
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+.search-form {
+  position: relative;
```
Comment (Ihor-Karpyn) [inline src/style.css:20]: Input will broke if u add some element to top of the form, let's add relative to the input wrapper,

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 4]
```diff
@@ -11,6 +11,37 @@
...
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      class="search-form"
+      action="/"
+      method="get"
+    >
+      <label class="search-input">
```
Comment (Ihor-Karpyn) [inline src/index.html:20]: Because it's not the input

```suggestion
      <label class="search-input-wrapper">
```

**Review by Ihor-Karpyn — APPROVED** [review]
LGTM

**Pull request closed without merging**